engines/account/app/workers/account/signup_worker.rb

```diff
@@ -0,0 +1,13 @@
+module Account
+  class SignupWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low
+
+    def perform(signup_id)
+      signup = Signup.find_by(id: signup_id)
+
+      SignupMailer.confirm_email(signup).deliver
```

Comment:
I think is not big deal. We're on `4.1.7`. 
Big deal @edulan @enricostano ? :smile: